airflow/contrib/hooks/sqoop_hook.py

```diff
@@ -159,11 +158,12 @@ def _get_export_format_argument(self, file_type='text'):
                                    "'sequence', 'parquet' or 'text'.")
 
     def _import_cmd(self, target_dir, append, file_type, split_by, direct,
-                    driver):
+                    driver, extra_import_options):
```

Comment:
add the option to pass in extra import arguments that sqoop offers. There are many options that sqoop offers that aren't part of the SqoopOperator and there should be a way to pass those in.